### PR TITLE
Fixed missing SOURCES from package build pipeline

### PR DESCRIPTION
**Pull request by AhmedBM**
## Description

* Missing SOURCES from package build pipeline artifact

## Checklist

- [X] I have read the [contribution guidelines](https://github.com/Azure/azure-osconfig/blob/main/CONTRIBUTING.md).
- [X] I added unit-tests to validate my changes. All unit tests are passing.
- [X] I have merged the latest `main` branch prior to this PR submission.
- [X] I submitted this PR against the `main` branch.

### CODE REVIEW

**Review by MariusNi — APPROVED** [review]
:shipit:

File: devops/pipeline/stages/source_build.yaml [hunk 1]
```diff
@@ -56,10 +56,10 @@ stages:
       displayName: Generate build
 
     - script: |
-        docker exec $(CONTAINER_ID) bash -c "cmake --build /AzOsConfig/build && cmake --build /AzOsConfig/build --target dist && mkdir /staging && cp -r /AzOsConfig/build/dist/* /staging"
+        docker exec $(CONTAINER_ID) bash -c "cd /AzOsConfig/build && cmake --build . && cmake --build . --target dist && mkdir staging && cp -r /AzOsConfig/build/dist/* staging"
```
Comment (robertschaedler3) [inline devops/pipeline/stages/source_build.yaml:59]: Is it possible to remove all instances of `AzOsConfig`? Will this break our pipelines to change it to `azure-osconfig`?

**Review by robertschaedler3 — APPROVED** [review]
👍🏻

File: devops/pipeline/stages/source_build.yaml (hunk 1 above)
Comment (AhmedBM) [reply devops/pipeline/stages/source_build.yaml:59]: i rather do that separately. Too many changes for just that reference change

**Pull request merged**